CONTRIBUTING.md

```diff
@@ -120,7 +122,105 @@ Once you are satisfied with your PR you should ask someone to review it. Before
 
-### Before the release
+### Local packaging test
+
+Before starting uploading things to pypi.org, there are a few tests one can perform locally.
+
+0. Make sure your local tests are all passing with environment variable `EXECUTE_TESTS_ON=master pytest` (see step Run tests locally above)
+
+1. Open a fresh python3 virtual environment and make sure you have the latest versions of setuptools and wheel installed:
+
+    ```bash
+    pip install --upgrade setuptools wheel twine
+    ```
+2. Move to the root of your local copy of this repository and prepare the python package and remove previous version distribution files with 
+    ```bash
+    python prepare_package.py
+    ```
+3. Install multi-vector-simulator form the local packages
+    ```bash
+    pip install -e .
+    ```
+4. Move outside of this repository (with `cd ..`)
+    ```bash
+    cd ..
+    ```
+5. Create a new empty folder
+    ```bash
+    mkdir empty_folder
+    ```
+6. Create a new empty folder
+    ```bash
+    mkdir empty_folder
+    ```
+7. Test the multi-vector-simulator default simulation
+    ```bash
+    mvs_tool
+    ```
+    It should run the simulation and save the results in the default output folder.
+    If the simulation does not run through, find out why and fix it.
+    If you run this local packaging test multiple times, either create a new empty folder or run `mvs_tool -f` to overwrite the default output folder.
+
+If this test passes locally, you can move to next step, upload a release candidate on pypi.org
+
+### Release candidate on pypi.org
+
+As on pypi.org, one is allowed to upload a given version only once, one need a way to test a release before making it official. This is the purpose of a release candidate.
+Technically, a release candidate is similar to a normal release in the sense that someone can `pip install` them. However users will know that the release candidate is only there for test purposes.
+
+1. Open a working python3 virtual environment and make sure you have the latest versions of setuptools and wheel installed:
+    ```bash
+    pip install --upgrade setuptools wheel twine
+    ```
+2. Make sure you pulled the latest version of `dev` branch from `origin`: `git checkout dev`, `git pull origin`.
+3. Change the version (without committing) with release candidates (add `rc1` to the `version_num`, for example `vX.Y.Zrc1`) before the actual release, as a release with a specific version number can only be uploaded once on pypi.
+4. Move to the root of your local copy of this repository and prepare the python package and remove previous version distribution files with 
+    ```bash
+    python prepare_package.py
+    ```
+    The last two lines should show the result of the twine check:
+    ```
+    Checking dist/multi_vector_simulator-X.Y.Zrci-py3-none-any.whl: PASSED
+    Checking dist/multi-vector-simulator-X.Y.Zrci.tar.gz: PASSED
+    ```
+    If one of the two is not `PASSED`, find out why and fix it.
```

Comment:
How do we fix things - we did not create a release branch jet, so I suppose we are fixing by fixing the dev branch? Maybe make it explicit that those fixes should be merged into dev first?